platform/mbed_retarget.cpp

```diff
@@ -126,6 +126,7 @@ void remove_filehandle(FileHandle *file) {
 #if DEVICE_SERIAL
 extern int stdio_uart_inited;
 extern serial_t stdio_uart;
+#endif
```

Comment:
The calls below depend on serial, not sure if we can close the `#if` here.

Reply:
Oh, this is not expected, I think its some sort of rebase/merge issue. Ill address it.